spec/helpers/warning_helpers.rb

```diff
@@ -0,0 +1,9 @@
+def silence_warnings
```

Comment:
This seems to just be a duplicate of the same method name in `/tests/test_helper.rb`? If so, we probably only need one or the other?

Reply:
Agreed. I wasn't sure what the state of the world was between spec/tests but it looks like spec is the newer thing. I'll keep the helper in spec and have tests require that one.

Reply:
Yeah, we should probably more explicitly document this. We started a transition toward spec and lost momentum, but the intention was still to land there in the end. At some point we will hopefully get back to that migration and out of this in-between state.